Scripts/Sources/Automation/Eval.swift

```diff
@@ -0,0 +1,329 @@
+import Foundation
+import PathKit
+import xcproj
+
+class Eval {
+    static func main() {
+        print("💁🏻‍♂️ Job type: \(TravisCI.jobType().description)")
+
+        if isSpecificJob() {
+            return
+        }
+
+        if TravisCI.isPullRquestJob() || Shell.nextArg("--env") == "pr" {
+            runPullRequestLane()
+        } else {
+            runContinousIntegrationLane()
+        }
+    }
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tevelee/Eval/blob/fc973589729700ac4c55183b359a3f3047c6157f/Scripts/Sources/Automation/Eval.swift#L19"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>